justfile

```diff
@@ -233,8 +233,9 @@ run *args:
     pathname=`just executable-path`
     ${pathname} {{ args }}
 
-# Execute unit tests.
-test *args=("./..."):
+# Execute unit tests. -short disables our functional tests
+# To run all of the tests, use `just test ./...`
+test *args=("-short ./..."):
```

Comment:
Maybe for a different pull request, but some tooling to call the functional tests directly could be nice.